src/graph.rs

```diff
@@ -1,4 +1,4 @@
 pub mod ts;
 pub mod sg;
 pub mod dg;
-pub mod lavatrice;
+// pub mod lavatrice;
```

Comment:
This was disabled because it was using `Path`, which has been removed. All code is now based on `PartialPath`.